lib/Controller/ConfigController.php

```diff
@@ -90,8 +94,15 @@ public function setConfig(array $values): DataResponse {
 	 * @param array<string,string> $values key/value pairs to store in app config
 	 * @return DataResponse
 	 */
+	#[Http\Attribute\PasswordConfirmationRequired]
```

Comment:
You need to use `@nextcloud/password-confirmation` in the frontend like in https://github.com/nextcloud/integration_zammad/blob/main/src/components/AdminSettings.vue#L125-L143

Reply:
forgot to push that :see_no_evil: